microsoft-edge/web-platform/ad-selection-api.md

```diff
@@ -0,0 +1,225 @@
+
+* `"ownership_token":` is the OT token generated for your individual domain registration.
+
+
+<!-- ====================================================================== -->
+## Deploy the Ad Selection API services
+
+The Ad Selection API uses a [trusted execution environment (TEE)](https://confidentialcomputing.io/wp-content/uploads/sites/85/2021/03/confidentialcomputing_outreach_whitepaper-8-5x11-1.pdf) to provide a level of assurance for data integrity, data confidentiality, and code integrity.  Services provided by the Ad Selection API must run in a TEE to secure the data used by these services.
+
+Ad Selection Services running in a TEE should be deployed on a cloud platform that supports the necessary security features.  Initially, services can be deployed in Azure using [Confidential ACI containers](/azure/container-instances/container-instances-confidential-overview).
+
+The Ad Selection API provides different services that need to be deployed by buyers or sellers.  Microsoft provides an image for each service, which can be deployed on a cloud provider.
+
+
+<!-- ------------------------------ -->
+#### Seller's Service
+
+* **SellerFrontEnd Service:** The front-end service provides a gRPC endpoint `/SelectAd`, which receives requests from the seller's ad service to initiate the Protected Audience auction flow.
+
+* **Auction Service:** This service provides a gRPC endpoint `/ScoreAds`, which receives requests from the **SellerFrontEnd** service with bids participating in the auction.  The service responds with a score value that the **SellerFrontEnd** uses to choose the winner.
+
+* **Seller's Key/Value Service:** This service receives requests from the **SellerFrontEnd** with lookup keys from buyers' bids (such as ad_render_urls) and returns real-time scoring signals required for the auction.  The Key/Value service will be run in Bring Your Own Service (BYOS) mode, so the seller does not need to deploy it in a TEE, and can instead use the image provided by Microsoft.
+
+
+<!-- ------------------------------ -->
+#### Buyer's Service
+
+* **BuyerFrontEnd Service:** The front-end service provides a gRPC endpoint `/GetBids`, which receives requests from **SellerFrontEnd** to initiate the bidding flow.
+
+* **Bidding Service:** This service provides an endpoint `/GenerateBids`, which receives requests from the **BuyerFrontEnd** service to handle the bidding and generate a bid.  The service generates a bid, chooses the winner, and selects the banner to be rendered.
+
+* **Buyer's Key/Value Service:** A buyer's Key/Value service receives requests from the **BuyerFrontEnd** service and returns real-time buyer data required for bidding, corresponding to lookup keys from Interest Groups.  This request happens once per workflow.  The Key/Value service will be run in BYOS mode.
```

Comment:
```suggestion
* **Buyer's Key/Value Service:** A Buyer's Key/Value Service receives requests from the **BuyerFrontEnd** service and returns real-time buyer data required for bidding, corresponding to lookup keys from Interest Groups.  This request happens once per workflow.  The Key/Value service will be run in BYOS mode.
```
is it possible to have consistent capzn throughout .md file, identical name of service everywhere?

Reply:
here's formatted as h6 headings:
```
<!-- ------------------------------ -->
#### Seller's Service


<!-- ---------- -->
###### SellerFrontEnd Service

The front-end service provides a gRPC endpoint `/SelectAd`, which receives requests from the seller's ad service to initiate the Protected Audience auction flow.


<!-- ---------- -->
###### Auction Service

This service provides a gRPC endpoint `/ScoreAds`, which receives requests from the **SellerFrontEnd** service with bids participating in the auction.  The service responds with a score value that the **SellerFrontEnd** uses to choose the winner.


<!-- ---------- -->
###### Seller's Key/Value Service

This service receives requests from the **SellerFrontEnd** with lookup keys from buyers' bids (such as ad_render_urls) and returns real-time scoring signals required for the auction.  The Key/Value service will be run in Bring Your Own Service (BYOS) mode, so the seller does not need to deploy it in a TEE, and can instead use the image provided by Microsoft.
```

Reply:
formatted as table
* enables omit the starting phrase in col 2 & omit "service" in col 1 - compact.
```
<!-- ------------------------------ -->
#### Seller services

| Service | Description |
| --- | --- |
| **SellerFrontEnd** | Provides a gRPC endpoint `/SelectAd`, which receives requests from the seller's ad service to initiate the Protected Audience auction flow. |
| **Auction** | Provides a gRPC endpoint `/ScoreAds`, which receives requests from the **SellerFrontEnd** service with bids participating in the auction.  The service responds with a score value that the **SellerFrontEnd** uses to choose the winner. |
| **Key/Value** | Receives requests from the **SellerFrontEnd** with lookup keys from buyers' bids (such as ad_render_urls) and returns real-time scoring signals required for the auction.  The Key/Value service will be run in Bring Your Own Service (BYOS) mode, so the seller does not need to deploy it in a TEE, and can instead use the image provided by Microsoft. |
```
rendered result:
![image](https://github.com/user-attachments/assets/fbde70f2-3bd7-428f-aa3d-274bbe115321)